src/features/user/Account/History.tsx

```diff
@@ -68,20 +126,33 @@ export default function History({
       <div className={'profilePageSubTitle'}>{t('me:donationsSubTitle')}</div>
       <div className={styles.pageContainer}>
         <div className={styles.filterRow}>
-          {accountingFilters &&
-            Object.entries(accountingFilters).map((item) => {
-              return (
-                <div
-                  key={item[0]}
-                  className={`${styles.filterButton} ${
-                    filter === item[0] ? styles.selected : ''
-                  }`}
-                  onClick={() => handleSetFilter(item[0])}
-                >
-                  {t(item[0])}
-                </div>
-              );
-            })}
+          {accountingFilters && (
+            <>
+              {Object.entries(accountingFilters).map((item) => {
+                return (
+                  <div
+                    key={item[0]}
+                    className={`${styles.filterButton} ${
+                      filter === item[0] ? styles.selected : ''
+                    }`}
+                    onClick={() => handleSetFilter(item[0])}
+                  >
+                    {t(item[0])}
+                  </div>
+                );
+              })}
+            </>
+          )}
+        </div>
+        <div
+          className={`${styles.issueButtonMobile}`}
+          onClick={!isLoading ? handleIssueReceipts : undefined}
+        >
+          {isLoading ? (
+            <CircularProgress color="inherit" size={20} />
+          ) : (
+            t('me:issueReceipts')
+          )}
         </div>
```

Comment:
I understand that the code above used `div` elements for buttons, but that should be avoided for semantic and accessibility reasons. The current implementation does not allow buttons to be tab selected via keyboard, among other things.

It would be better to use a `button` element here (either regular html, or MUI).

Reply:
Fixed here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1677/commits/bff0d19a7d796e72ba4a2d6bb4d5cd7896e8778d